packages/foundation/src/atoms/roles/icon.css

```diff
@@ -0,0 +1,24 @@
+html * {
+    /* LIGHT */
+    --o-ui-icon-1-light: var(--o-ui-marine-900);
+    --o-ui-icon-2-light: var(--o-ui-marine-500);
+    --o-ui-icon-primary-light: var(--o-ui-primary-500);
+    --o-ui-icon-danger-light: var(--o-ui-alert-500);
+    --o-ui-icon-danger-strong-light: var(--o-ui-alert-900);
+    --o-ui-icon-positive-light: var(--o-ui-green-500);
+    --o-ui-icon-positive-strong-light: var(--o-ui-green-900);
+    --o-ui-icon-warning-light: var(--o-ui-sunray-500);
+    --o-ui-icon-warning-strong-light: var(--o-ui-sunray-900);
+    --o-ui-icon-info-light: var(--o-ui-marine-500);
+    /* DARK */
+    --o-ui-icon-1-dark: var(--o-ui-static-white);
+    --o-ui-icon-2-dark: var(--o-ui-cloud-500);
+    --o-ui-icon-primary-dark: var(--o-ui-primary-700);
+    --o-ui-icon-danger-dark: var(--o-ui-alert-700);
+    --o-ui-icon-danger-strong-dark: var(--o-ui-alert-50);
```

Comment:
I am not sure about this "strong" and "pale" naming. It seem to go against our current naming strategy which is to use "1", "2", "3", etc....

I am not close to it but I do want to make sure our naming is consistent.

Reply:
at first I went light but light-light war a smell, pale felt wrong too, going with 1 - 2 is ok although the consistency wouldn't be there as not every colour has a "pale" version